src/Matter/Artwork.elm

```diff
@@ -0,0 +1,425 @@
+            ]
+            []
+        , Html.figcaption
+            [ T.text_center
+            ]
+            [ Html.h3
+                [ T.mt_2
+                , T.text_xl
+                , T.font_display
+                , T.text_gray_100
+                ]
+                [ Html.text name ]
+            , Html.p
+                [ T.mt_2
+                , T.font_body
+                , T.text_gray_300
+                , T.text_base
+                ]
+                description
+            , Html.p
+                [ T.mt_1
+                , T.font_body
+                , T.text_gray_300
+                , T.text_base
+                ]
+                [ Html.text "Art by "
+                , Html.span [ T.text_gray_100 ] [ Html.text author ]
+                ]
+            , Html.p
+                [ T.mt_1
+                , T.font_body
+                , T.text_gray_300
+                , T.text_base
+                ]
+                [ Html.text date ]
+            ]
+        ]
+
+
+
+-- CALL TO ACTION
+
+
+callToAction : PagePath -> Model -> CallToActionData -> Html Msg
+callToAction pagePath model data =
+    Html.div
+        [ T.bg_gray_600
+        , T.flex
+        , T.flex_col
+        , T.overflow_hidden
+        , T.px_6
+        , T.py_16
+        , T.md__py_24
+        ]
+        [ Html.div
+            [ T.mx_auto
+            , T.flex
+            , T.flex_col
+            , T.lg__flex_row
+            ]
+            [ Html.img
+                [ A.src (ImagePath.toString images.content.artworkPage.heroImage)
+                , A.width 844
+                , A.height 840
+                , T.flex_shrink
+                , T.max_w_sm
+                , T.px_6
+                , T.w_full
+                , T.mx_auto
+                ]
+                []
+            , Html.div
+                [ T.text_center
+                , T.lg__text_left
+                , T.lg__max_w_sm
+                , T.lg__my_auto
+                , T.lg__ml_20
+                ]
+                -- TODO move to data
+                [ Html.h3
+                    [ T.text_2xl
+                    , T.text_gray_100
+                    , T.font_display
+                    , T.md__text_3xl
+                    ]
+                    [ Html.text data.title ]
+                , Html.p
+                    [ T.mt_4
+                    , T.font_body
+                    , T.text_gray_300
+                    , T.md__text_lg
+                    ]
+                    data.body
+                , Html.button
+                    [ T.appearance_none
+                    , T.cursor_pointer
+                    , T.mt_8
+                    , T.text_gray_200
+                    , T.bg_gray_200
+                    , T.leading_relaxed
+                    , T.px_4
+                    , T.py_1
+                    , T.rounded_lg
+                    , T.text_gray_600
+                    , T.md__text_lg
```

Comment:
Ah that explains it.

Yeah, I've seen the 'Responsive' comments. Alright.

It's possible to create an elm-review rule for alphabetized class order :thinking: (Definitely way too much effort, but kinda cool)

Should we create a styleguide somewhere?

Reply:
An elm-review rule could be cool, but not sure how that'd would work (you'd have split up into sections based on comments?). Would be super cool if it could auto fix based on said created rule 💯 

We can do a small style guide for sure 👍 
Let me know if you'd like me to do that, or if you wanna do that.

Reply:
Let's put as much as possible on to @matheus23 -- and I'll just file it as an issue here for now and we can prioritize as we go.